packages/core/fs/src/index.js

```diff
@@ -4,9 +4,9 @@ import type {FilePath} from '@parcel/types';
 import path from 'path';
 
 export type * from './types';
-export {NodeFS} from './NodeFS';
-export {MemoryFS} from './MemoryFS';
-export {OverlayFS} from './OverlayFS';
+export * from './NodeFS';
+export * from './MemoryFS';
+export * from './OverlayFS';
```

Comment:
In most index files, you removed the namespace export but you added it in this case?

Reply:
Yeah... it's because we rely on these being executed immediately since they register types with the serializer. 😞